controllers/signupController.js

```diff
@@ -29,17 +25,26 @@ module.exports = {
         } catch(err){
             res.send({ err });
         }
+         
+        try { 
+            await bcrypt.hash(password, null, null, (err, hash) => {
```

Comment:
```
user.password = await bcrypt.hash(password, null, null);
```

Reply:
returns
{
    "err": {},
    "success": false
}

Reply:
is `bcrypt.hash` an async function

Reply:
yes, it is